spec/scripts/append_last_edited_by_spec.rb

```diff
@@ -23,17 +23,19 @@ def trigger_automation(post)
   context "#trigger!" do
     it 'works for newly created post' do
       cooked, updated_at, date_time = trigger_automation(post)
-      expect(cooked.ends_with?("<blockquote>\n<p>Last edited by #{post.username} <span data-date=\"#{updated_at.to_date.to_s}\" data-time=\"#{updated_at.strftime("%H:%M:%S")}\" class=\"discourse-local-date\" data-timezone=\"UTC\" data-email-preview=\"#{date_time} UTC\">#{date_time}</span></p>\n</blockquote>\n</div>")).to be_truthy
+      expect(cooked.include?("<blockquote class=\"discourse-automation\">")).to be_truthy
+      expect(cooked.include?("Last edited by <a class=\"mention\" href=\"/u/#{post.username}\">@#{post.username}</a>")).to be_truthy
```

Comment:
This will likely break whenever the translation text changes

Reply:
I fixed it in the [other PR](https://github.com/discourse/discourse-automation/pull/124). So it's removed from here.